core/admin/mailu/internal/nginx.py

```diff
@@ -81,6 +81,13 @@ def handle_authentication(headers):
         raw_password = urllib.parse.unquote(headers["Auth-Pass"])
         password = raw_password.encode("iso8859-1").decode("utf8")
         ip = urllib.parse.unquote(headers["Client-Ip"])
+        port = int(urllib.parse.unquote(headers["Auth-Port"]))
```

Comment:
I think this causes the test to fail. Here you overwrite the port (see line 74) of the destination postfix server. 
Port 10025 is overwritten with port 587 (the port from the header Auth-Port.)

This results in the connection refused error that nginx receives when it tries to proxy the connection to port 587 instead of 10025.

I suggest to rename the var port to something else and of course also rename the var on line 85 (if port == 25).

Reply:
Doh.
Thanks!